### PR TITLE
review js array methods concat & valueOf

### PR DESCRIPTION
**Pull request by yangc95**
### Type of Change

- [ ] Adding a new entry
- [x] Editing an existing entry (fixing a typo, bug, issues, etc)
- [ ] Updating the documentation

### Checklist

- [x] My entry follows the Codecademy Docs style guide
- [x] I have performed a self-review of my own writing and code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my entry and corrected any misspellings
- [x] I have confirmed my changes are not being pushed from my forked `main` branch
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making
- [x] All writings are my own

### CODE REVIEW

**Review by EugeneGohh — APPROVED** [review]
LGTM.

**Review by Dusch4593 — APPROVED** [review]
LGTM!

File: content/javascript/concepts/arrays/terms/valueOf/valueOf.md [hunk 1]
```diff
@@ -12,16 +12,14 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Returns a new array with all the items of the original array.
+Returns the value of all the elements of the original array.
```
Comment (sonnynomnom) [inline content/javascript/concepts/arrays/terms/valueOf/valueOf.md:15]: The `Description` metadata can start with the verb. 

For these though, let's do:

The `.valueOf()` method returns the value of all the elements of the original array.

File: content/javascript/concepts/arrays/terms/valueOf/valueOf.md [hunk 2]
```diff
@@ -33,3 +31,5 @@ const newArray = programmingLanguages.valueOf();
 console.log(newArray);
 // Output: ['JS', 'Python', 'Java', 'C++']
 ```
+
+**Note:** This method will not change the original array. However, if the original array or the variable which holds it (via the `valueOf` method) is manipulated, the value of the array will change accordingly.
```
Comment (sonnynomnom) [inline content/javascript/concepts/arrays/terms/valueOf/valueOf.md:35]: (using the `.valueOf()` method)

File: package.json [hunk 3]
```diff
@@ -5,13 +5,19 @@
     "glob": "^7.1.7"
   },
   "devDependencies": {
-    "@codecademy/eslint-config": "5.0.0",
+    "@codecademy/eslint-config": "^3.1.0",
```
Comment (sonnynomnom) [inline package.json:8]: This file shouldn't be changed, right?

**Review by sonnynomnom — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (yangc95) [reply package.json]: removed (along with the yarn file)!

**Review by sonnynomnom — APPROVED** [review]

**Comment by sonnynomnom** [comment]
merged!

**Comment by sonnynomnom** [comment]
Made two tiny changes: 

https://github.com/Codecademy/docs/commit/aa57adcda872e6599656860dc1558a728fa6d526

**Pull request merged**